### PR TITLE
Alert transaction integration

### PR DESCRIPTION
**Pull request by kj-crypto**

### CODE REVIEW

File: electrum/synchronizer.py [hunk 1]
```diff
@@ -50,11 +50,29 @@ def history_status(h):
...
         return None
     status = ''
-    for tx_hash, height, *__ in h:
-        status += tx_hash + ':%d:' % height
+    for tx_hash, height, tx_type in h:
+        status += f'{tx_hash}:{height:d}:{tx_type:s}:'
     return bh2u(hashlib.sha256(status.encode('ascii')).digest())
 
 
+def filer_out_double_atxid_from_history(raw_history) -> list:
```
Comment (harzo) [inline electrum/synchronizer.py:58]: typo: `filer`
btw: i would name it `get_unique_atxid_form_history` 
filter sounds more like operation `in place`

Comment (kj-crypto) [reply electrum/synchronizer.py]: done :)

File: electrum/synchronizer.py [hunk 2]
```diff
@@ -195,7 +195,7 @@ async def _on_address_status(self, addr, status):
             item['height'],
             item.get('tx_type', TxType.NONVAULT.name)
         ), result))
-        filtered_history = filer_out_double_atxid_from_history(hist)
+        filtered_history = get_unique_atxid_form_history(hist)
```
Comment (harzo) [inline electrum/synchronizer.py:198]: Dont want to be picky but... typo ->  `form`

**Comment by harzo** [comment]
Good to go :+1:

**Pull request merged**